### ISSUE TITLE
Dev: fix eslint rules

### ISSUE THREAD

**Issue by avaer** [webaverse/app#1432]
We have a https://github.com/webaverse/app/blob/master/.eslintrc.json, but some of the rules are in disagreement with the main code style.

This multi-line if enforcement should be disabled:

```
if (
   this.dialogClient._protoo._transport._ws.readyState === WebSocket.OPEN
) {
```

And the ternary mode needs to be changed to mirror if statements when it is too long:

```
cond ?
'yes'
:
'no'
```

**Comment by avaer** [comment]
Once this is done we can do a cleanup pass over the codebase in a separate PR.
